src/Lib/Modes/PathEditMode.vala

```diff
@@ -192,7 +314,11 @@ public class Akira.Lib.Modes.PathEditMode : AbstractInteractionMode {
     public override bool motion_notify_event (Gdk.EventMotion event) {
         Geometry.Point point = Geometry.Point (event.x, event.y);
 
-        if (!is_edit_path) {
+        if (is_edit_path) {
+            if (!edit_model.selected_pts.is_empty && is_click) {
+                // If user selected a point and clicked and dragged it, change its position.
+                edit_model.modify_point_value (point);
```

Comment:
we are not snapping to pixel grid. And after adding the second point, the whole thing moves.

Reply:
@mbfraga Can you elaborate as to what the problem is? I haven't worked with pixel grid yet. So I don't understand what exactly you want me to do here.

Reply:
Akira works mostly by abiding to the pixel grid. The core idea is that anything you add will generally be snapped to the pixel grid--which makes it good for UI design (figma). So when you add points to a path, it should always be choosing the best pixel position, and not the arbitrary location that was clicked. 

For better understanding--go to figma and zoom at the maximum level and add a path and see the positions where you are actually allowed to add points to. Now do the same in Akira to see the difference (enable the grid view in Akira--maybe it's the default?)

Reply:
This is still not happening.